lib/matplotlib/tests/test_colors.py

```diff
@@ -156,6 +156,193 @@ def test_LogNorm():
     assert_array_equal(ln([1, 6]), [0, 1.0])
 
 
+def test_FuncNorm():
```

Comment:
But assert doesn't spit error messages, so you're kind of left guessing what specific thing that assert was trying to test.
http://doc.pytest.org/en/latest/parametrize.html and [test_category.py](https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/tests/test_category.py) has lots of mpl examples.